src/actions/actionClipboard.tsx

```diff
@@ -131,10 +131,17 @@ export const actionCopyAsPng = register({
 export const copyAllTextNodesAsText = register({
   name: "copyAllTextNodesAsText",
   trackEvent: { category: "element" },
-  perform: (elements) => {
-    const text = (
-      getNonDeletedElements(elements) as ExcalidrawTextElement[]
-    ).reduce((acc, element) => `${acc}${element.text}\n`, "");
+  perform: (elements, appState) => {
+    const selectedElements = getSelectedElements(
+      getNonDeletedElements(elements),
+      appState,
+      true,
```

Comment:
this should be `false` since we are not including bound text elements

Reply:
That's what I thought!

Reply:
Earlier since we decided to show only for text elements that's where I thought it could cause confusion for bound text, but now we are showing for mixed elements so let's include bound text too